src/ui/common/src/components/workflows/workflowHeader.tsx

```diff
@@ -93,6 +94,20 @@ const WorkflowHeader: React.FC<Props> = ({ user, workflowDag, workflowId }) => {
     );
   }
 
+  let airflowUpdateWarning = null;
+  if (workflowDag.engine_config.type == EngineType.Airflow 
+    && !workflowDag.engine_config.airflow_config?.matches_airflow) {
+      airflowUpdateWarning = (
+        <Box>
```

Comment:
@agiron123 @vsreekanti What's the best way to make this box's width smaller. It takes up almost the entire page right now.

Reply:
I would just set a max width in pixels on the box.

Reply:
Yeah, something like maxWidth="800px" would be good to have here. @saurav-c